pkg/common/common.go

```diff
@@ -206,17 +211,34 @@ func AppendTTMapAsCopy[T comparable](add ...map[T]T) map[T]T {
 	return base
 }
 
-// CCOWorkflow checks if the AWS STS secret is to be obtained from Cloud Credentials Operator (CCO)
+// CCOWorkflow checks if the AWS STS secret or GCP WIF is to be obtained from Cloud Credentials Operator (CCO)
 // if the user provides role ARN during installation then the ARN gets set as env var on operator deployment
-// during installation via OLM
+// during installation via OLM. Similarly, if the user provides AUDIENCE and SERVICE_ACCOUNT_EMAIL for GCP WIF,
+// these are set as env vars on the operator deployment.
```

Comment:
did not understand this comment. Are not these already env vars?

Reply:
No. We create CredentialsRequest later with info user input during install exposed to us via envvar. These are not themselves credentials.